Graphir.API/Extensions/Startup.Device.cs

```diff
@@ -14,6 +14,14 @@ public static IRequestExecutorBuilder AddDevice
             return graphBuilder
                 .AddTypeExtension<ResourceQuery<Hl7.Fhir.Model.Device, DeviceType>>()
                 .AddDataLoader<ResourceByIdDataLoader<Hl7.Fhir.Model.Device>>()
+                .AddType<DeviceParentType>()
```

Comment:
Use the `.Type<>` extension on the ObjectType descriptor to automatically register dependent nested types. That way we can reduce the number of classes needing to be added to the startup.

Reply:
Type extension for primitive types removed and unnecessary object type registration removed in the startup class